### PR TITLE
Fix/build hook

### PR DESCRIPTION
**Pull request by IvanHdzC**
This PR fixes docker hub building failures and pushes the two new images into the repository.

### CODE REVIEW

File: docker/hooks/post_push [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with iotagent-json. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact
+# with iot_support at tid dot es
+
+docker image push $IMAGE_NAME-pm2 
+
+docker image push $IMAGE_NAME-distroless
```
Comment (fgalan) [inline docker/hooks/post_push:25]: I understand that the default image (the one for node-slim) is automatically pushed so there is no need of including it in posst_push. Is my understanding correct?

Comment (IvanHdzC) [reply docker/hooks/post_push:25]: That's correct @fgalan. The tagged version is pushed by default. So there is no need to push it again.

Comment (fgalan) [reply docker/hooks/post_push:25]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**